fastscroll/build.gradle

```diff
@@ -38,6 +38,7 @@ android {
 }
 
 dependencies {
+    api 'androidx.appcompat:appcompat:1.0.2'
```

Comment:
Adding this is unnecessary since appcompat is already imported as a transitive dependency.